deepchem/feat/protein_tokenizer.py

```diff
@@ -0,0 +1,155 @@
+from deepchem.feat.base_classes import Featurizer
+from typing import Iterable
+import numpy as np
+import logging
+logger = logging.getLogger(__name__)
+
+# Integer tokens for every amino acid code in the FASTA format.
+FASTA_tokens = {
+    "A": 0,
+    "B": 1,
+    "C": 2,
+    "D": 3,
+    "E": 4,
+    "F": 5,
+    "G": 6,
+    "H": 7,
+    "I": 8,
+    "J": 9,
+    "K": 10,
+    "L": 11,
+    "M": 12,
+    "N": 13,
+    "O": 14,
+    "P": 15,
+    "Q": 16,
+    "R": 17,
+    "S": 18,
+    "T": 19,
+    "U": 20,
+    "V": 21,
+    "W": 22,
+    "Y": 23,
+    "Z": 24,
+    "X": 25,
+    "*": 26,
+    "-": 27
+}
+
+
+class ProteinTokenizer(Featurizer):
+  """Tokenizes protein sequences in FASTA-format with start and end annotations
+  into integer arrays.
+
+  This tokenizer takes in an array of FASTA-format protein sequence strings
+  that are terminated on both ends with [CLS] and [SEP] annotations, and
+  converts each string into a matching array of integer tokens.
+
+  The tokenizer is case-insensitive and safely ignores all non-FASTA characters
+  (spaces, newlines, commas, etc.).
+
+  eg. ["EMV[CLS]ABCDE[SEP]X", "WMY[CLS]FGH[SEP]XAB"] -> [[0, 1, 2, 3, 4], [5, 6, 7]]
```

Comment:
Can you add a usage example here in the docstring?

Reply:
Line 51 is supposed to be one... Is there a specific format that I should abide by?